src/app/cube/content-pages/contribute-page/contribute-page.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { sections } from './copy';
+
+@Component({
+  selector: 'clark-contribute-page',
+  templateUrl: './contribute-page.component.html',
+  styleUrls: ['./contribute-page.component.scss']
+})
+
+export class ContributePageComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void { }
+  }
+
+  title = 'Contribute';
+   copy = sections;
```

Comment:
Why are we setting a variable to another variable here?

Reply:
For lines 17 and 18, the reason is because how the copy.ts file communicates to the contribute.html file through using the component.ts file. In the same way, the html file can only communicate through the component.ts file.

So long story short, the component.ts file acts as a connector for the contribute.html and copy.ts files and that is why copy is set to the section. If there's a better way to do this, I'd be open to suggestions.